src/template/partials/footer.html

```diff
@@ -1,3 +1,52 @@
 <footer>
+  <!-- Start Footer -->
+  <div class="footer">
+  
+    <div class="container">
+        <div class="download">
+            <h3>Grow your business fast <br>with Sark SaaS Landing.</h3>  
+                <button name="Download" value="Download">Download Now</button>
+        </div>
+        <div class="links">
```

Comment:
![Capture](https://github.com/shaban011147/landing/assets/106445705/df73d131-2ead-4273-b427-ea9e363f82cc)
space between this links and div (downloads) not equal 80px but equal 86px

Reply:
done